grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -58,6 +59,7 @@ export class ClientPageStore extends ComponentStore<ClientPageState> {
       private readonly configFacade: ConfigFacade,
   ) {
     super({
+      clientId: '',
```

Comment:
Here, `clientId` has a special value `''` that indicates no client has been selected. Please change the type of `clientId` to `string | undefined`, e.g. by adding a `?` to the variable definition. Then, use `undefined` as value instead of the empty string. Ultimately, you can remove this line, because a key that does not exist is `undefined` in TS.

Reply:
That's better, thanks!